internal/exec/help.go

```diff
@@ -20,43 +19,59 @@ func processHelp(
 		u.PrintMessage(fmt.Sprintf("atmos %s <command> <component> -s <stack> [options]", componentType))
 		u.PrintMessage(fmt.Sprintf("atmos %s <command> <component> --stack <stack> [options]", componentType))
 
-		if componentType == "terraform" {
-			u.PrintMessage("\nAdditions and differences from native terraform:")
-			u.PrintMessage(" - before executing other 'terraform' commands, 'atmos' runs 'terraform init'")
-			u.PrintMessage(" - you can skip over atmos calling 'terraform init' if you know your project is already in a good working state by using " +
-				"the '--skip-init' flag like so 'atmos terraform <command> <component> -s <stack> --skip-init")
-			u.PrintMessage(" - 'atmos terraform deploy' command executes 'terraform apply -auto-approve' (sets the '-auto-approve' flag when running 'terraform apply')")
-			u.PrintMessage(" - 'atmos terraform deploy' command supports '--deploy-run-init=true/false' flag to enable/disable running 'terraform init' " +
-				"before executing the command")
-			u.PrintMessage(" - 'atmos terraform apply' and 'atmos terraform deploy' commands support '--from-plan' flag. If the flag is specified, " +
```

Comment:
We need to make sure `--from-plan` is documented

Reply:
This is done